cmake/cmake_uninstall.cmake.in

```diff
@@ -0,0 +1,24 @@
+# This uninstall script is copied from the CMake wiki page:
+# https://cmake.org/Wiki/CMake_FAQ#Can_I_do_.22make_uninstall.22_with_CMake.3F
+
+if(NOT EXISTS "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
+  message(FATAL_ERROR "Cannot find install manifest: @CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
+endif()
+
+file(READ "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt" files)
+string(REGEX REPLACE "\n" ";" files "${files}")
```

Comment:
Also, if you are going to change the delimiter from `\n` to `NUL`, please _do not call it_ `*.txt`.  It is no longer a human-readable file, which I think was the intent of the `.txt` generation in the first place.

I don't feel too badly about not handling the newline path case though.  I've never seen it in the wild and I'm pretty sure that a whole lot of things are going to break if it happens.  I would prefer a more readable `manifest.txt` that is also handy to pipe into various shell tools.